hackclub.com.yaml

```diff
@@ -1272,8 +1272,26 @@ conduct-test:
   value: v=spf1 include:amazonses.com ~all
 conduct:
 - ttl: 300
-  type: CNAME
+  type: ALIAS
```

Comment:
Alias needs to become an A record then

Reply:
Per https://github.com/octodns/octodns/blob/main/docs/records.rst#record-level-lenience, if it does need to be an FQDN and not an IP, can set the lenient flag